core/templates/dev/head/components/ExplorationSummaryTileDirective.js

```diff
@@ -90,7 +90,8 @@ oppia.directive('explorationSummaryTile', [function() {
 
         $scope.avatarsList = [];
         $scope.contributors.forEach(function(contributorName) {
-          var DEFAULT_PROFILE_IMAGE_PATH = '/images/avatar/user_blue_72px.png';
+          var DEFAULT_PROFILE_IMAGE_PATH = ('/static/' + GLOBALS.CACHE_SLUG +
```

Comment:
Instead of doing basically the same thing all over the place, I suggest adding a new method to UrlInterpolationService, named something like getStaticImageUrl. You'd pass in '/avatar/user_blue_72px.png' and it would return '/static' + GLOBALS.CACHE_SLUG + '/images' + imagePath. The method should check that the path that's passed in starts with a '/', and there should be tests ensuring that it does.